### DISCUSSION TITLE
Typo / error in wiki/Installation

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#594] [General]
Line ~5
```
$ sudo dpkg -i opensnitch*.deb python3-opensnitch-ui*.deb; sudo apt -f install
```
apt -f ?

**Comment by gustavo-iniguez-goya** [comment]
Yeah, `apt -f` or `apt-get -f` (--fix-broken) to resolve missing dependencies.

Installing with dpkg doesn't resolve the needed dependencies.

**Reply by BobSquarePants** [reply]
Once again thank you @gustavo-iniguez-goya .

I asked because **apt -f** don't work for me (MX Linux) 
Neither when I checked with : https://explainshell.com/explain?cmd=apt+-f

But indeed
https://explainshell.com/explain?cmd=apt-get+-f
is correct, may be you should write apt-get -f in the line 5 ?

**Reply by BobSquarePants** [reply]
@gustavo-iniguez-goya, I have done some research and indeed `apt -f install` work's..

I'll contact explainshell.com
